controllers/listings.js

```diff
@@ -23,4 +23,6 @@ listingRouter.post('/', async (request, response) => {
     }
 })
 
-module.exports = listingRouter
\ No newline at end of file
+module.exports = listingRouter
+
+// test comment
```

Comment:
Dehä e skit

Reply:
I would suggest to move the documentation comment to the top of the page and explain a bit what the code really does.